src/js-marketing-notification.ts

```diff
@@ -30,17 +31,22 @@ export class JsMarketingNotification {
   }
 
   private isFirstPageVisit(): boolean {
-    return null === localStorage.getItem(this.popupTitle);
+    return null === localStorage.getItem(this.notificationTitle);
   }
 
-  private createNotificationTab(container: Element): void {
-    const tab = document.createElement('div');
-    tab.classList.add('marketing-notification__tab');
-    tab.addEventListener('click', e => {
+  private createNotificationTab(container: Element, content: string): void {
```

Comment:
I might make this an optional param by giving it a default value. You decide. No right or wrong answer.